src/components/main/compose/messages/mod.rs

```diff
@@ -399,6 +406,14 @@ pub fn Messages(cx: Scope<Props>) -> Element {
                                     Err(_) => { rsx!{ span { "Something went wrong" } } }
                                 }
                             }),
+                            (message_id == first_unread_message_id).then(||
```

Comment:
Are we still conditionally rendering this way? I remember hearing we aren't doing inline conditional rendering? Or maybe I miss-heard, but either way if it's not a big performance drop it can be cleaned up later. @InfamousVague?